divi/src/miner.cpp

```diff
@@ -169,8 +169,7 @@ CBlockTemplate* CreateNewBlock(const CScript& scriptPubKeyIn, CWallet* pwallet,
 
         SetRequiredWork(*pblock);
 
-        pblock->nTime = GetAdjustedTime();
-        int64_t nSearchTime = pblock->nTime; // search to current time
+        int64_t nSearchTime = SetBlockTime(*pblock); // search to current time
```

Comment:
Too small an extraction at this specific point, but will come in handy for other stuff later

Reply:
Also, if it's a setter function you don't want setter functions returning anything other than maybe a bool.